docs/spec_draft.md

```diff
@@ -931,6 +932,55 @@ spaces of `result` and `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reverse
+
+### Semantics
+
+Reverses the order of elements in the `operand` along the specified 
```

Comment:
Overall lgtm. Some changes to match the wording of the rest of the spec. Wdyt?

```
Reverses the order of elements in the `operand` along the specified `dimensions`
and produces a `result` tensor. Each element of `operand` tensor at each index
is stored into the `result` tensor such that the indices at `dimensions[k]` of
`operand` elements are reversed for all dimensions `k` in `dimensions`. More
formally, `result[i0, ..., ik,..., iR-1] = operand[i0, ..., ik',..., iR-1]`
where `ik + ik' = dk - 1` for all dimensions `k` in `dimensions`.
```

Reply:
Reverses the order of elements in the `operand` along the specified `dimensions`
and produces a `result` tensor.  For every dimension `k` in `dimensions`, an element at index `[i0, ..., ik,..., iR-1]` is swapped with the element at index `[i0, ..., dk-ik-1,..., iR-1]`, for all elements of the operand, thus reversing along the dimension `k`. 
More formally, after reversing along all dimensions, `result[i0, ..., ik,..., iR-1] = operand[i0, ..., ik',..., iR-1]`
where `ik + ik' = dk - 1` for all dimensions `k` in `dimensions`.

Reply:
How about this.

Reply:
Can we drop the second sentence? I don't think it adds much for the first and third sentences, but the mental overhead is significant.

"Reverses the order of elements in the `operand` along the specified `dimensions` and produces a result tensor. More formally, `result[i0, ..., ik,..., iR-1] = operand[i0, ..., ik',..., iR-1]` where `ik + ik' = dk - 1` for all dimensions `k` in `dimensions`."